manage/trigger/how-do-they-work.rst

```diff
@@ -111,3 +111,25 @@ A trigger can do the following things once its conditions have been met:
    This allows you to help your agents with specific information if needed.
    (e.g. automated changes a trigger applied to the ticket)
 
+Localization of Trigger Based Notifications
+-------------------------------------------
+
+To translate the output of variables and to format date and time stamps for
+notifications in triggers, use the ``t`` and ``dt`` methods.
+The ``t`` method translates the output of variables according to the selected
+"Locale" field below.
+The ``dt`` method transforms a date and time stamp according to the selected
+"Timezone".
+
+.. figure:: /images/manage/trigger/notification-localization.png
+   :align: center
+   :scale: 90%
+   :alt: Screenshot showing localization section of trigger dialog
+
+Be aware that it is not possible to perform an automatic translation for
+different languages in one trigger. That means if you want to create
+notifications for different languages/regions, you have to create one trigger
+for each language and execute them based on user or organization attributes.
```

Comment:
"Best practice is to create single separated localized triggers for each language and execute them based on user or organization attributes."